Doc/whatsnew/3.8.rst

```diff
@@ -952,6 +952,12 @@ Build and C API Changes
   allocation or deallocation may need to be adjusted.
   (Contributed by Eddie Elizondo in :issue:`35810`.)
 
+* The new function :c:func:`PyCode_NewWithPosOnlyArgs` substitutes
+  :c:func:`PyCode_New` as the prefered way to create code objects, having an
+  extra parameter representing the number of positional only parameters
+  (**posonlyargcount**).  The old function is kept for backward compatibility
+  reasons and will set the number os positional only parameters to 0.
```

Comment:
I'm not sure about making PyCode_NewWithPosOnlyArgs the preferred way. PyCode_New() is fine for most use cases, no?